### PR TITLE
New Staff List

### PR DESCRIPTION
**Pull request by BlackGoat**
The purges and promotions

### CODE REVIEW

**Comment by Spongy** [comment]
Setting up PR reference, please wait...

**Comment by Zidane** [comment]
@BlackGoat 

Can you use the descriptions on my staff list?

**Comment by lukegb** [comment]
I personally think it would be polite to at the very least have a section for retired staff. I'm not sure it makes sense to give them their own page but it's probably best to recognize the contributions that people have made in the past.

**Comment by Zidane** [comment]
@lukegb 

Great idea. We should definitely do so.

File: source/about/staff.rst [hunk 1]
```diff
@@ -29,68 +35,56 @@ The **Team Leaders** of Sponge each coordinate a separate aspect of the Sponge p
...
-|                                   | DevOps and IRC Flag Lead          |
-+-----------------------------------+-----------------------------------+
-| sibomots                          | Developer                         |
-|                                   |                                   |
-|                                   | LLC Leader                        |
+| simon816                          | Developer                         |
 +-----------------------------------+-----------------------------------+
 | theresajayne                      | Developer                         |
 +-----------------------------------+-----------------------------------+
+| Voltasalt                         | Developer                         |
```
Comment (jamierocks) [inline source/about/staff.rst]: AzureousNation and the other Granite devs?

**Comment by BlackGoat** [comment]
@Zidane To which list do you refer? The staff list table on gdocs looks out of date. I got this update from the MineCon email draft.

File: source/about/staff.rst [hunk 2]
```diff
@@ -12,14 +12,20 @@ The **Project Leaders** are the stewards of the Sponge project, and make the fin
 +-----------------------------------+-----------------------------------+
 | Name                              | Role                              |
 +===================================+===================================+
-| blood                             | Implementation Development Lead   |
```
Comment (jamierocks) [inline source/about/staff.rst]: Should @Voltasalt or someone be here for the Implementation (Vanilla) Lead?

Comment (Zidane) [inline source/about/staff.rst]: @jamierocks 

I lead the Vanilla impl and API. Vanilla has become somewhat of my baby to work on :P.

Comment (jamierocks) [inline source/about/staff.rst]: Ah.

**Pull request merged**